app/assets/vue/views/Home.vue

```diff
@@ -1,19 +1,60 @@
 <template>
-  <section class="section">
-    <div class="content">
-      <h1 class="title is-1">
-        {{ titleLabel }}
-      </h1>
-    </div>
-
+  <section>
     <div class="columns">
       <div class="column is-8">
         <section class="section box">
-          <h2 class="subtitle">
-            {{ $t("home.welcome") }}
-          </h2>
+          <div class="mb-2 is-flex is-flex-direction-column is-justify-content-center">
+            <div class="mb-1">
+              <h1 class="title is-2">
+                {{ titleLabel }}
+              </h1>
+            </div>
+
+            <div>
+              <h2 class="subtitle">
+                {{ $t("home.welcome") }}
+              </h2>
+            </div>
+          </div>
+
+          <div class="user-datas-div">
```

Comment:
Data never takes a `s` in English
```suggestion
          <div class="user-data-div">
```